portfolio/src/main/webapp/projects.html

```diff
@@ -15,6 +17,9 @@
           <a class="navigation-bar-items" href="blog_posts.html">Blog Posts</a>
       </ul>
     <div id="content">
+        <h2>Google Charts Library</h2>
+        <p>Graphs made from the Google Charts Library</p>
+        <div id="chart-container"></div>

```

Comment:
Nit: Lets separate this block of code from the block below with a newline to help with readability.

Reply:
Added a newline.